### DISCUSSION TITLE
Effect lighting in AR

### DISCUSSION THREAD

**Discussion by MathDestruction** [google/model-viewer#2862] [Q&A]
Hi,
I was wondering whether there is a way to adjust the environment lighting (the estimated lighting) while in AR model?

**Comment by elalish** [comment]
Not currently, unless you don't use `xr-environment`, in which case any `environment-image` you specify will be used instead. Personally, I'm not very impressed with ARCore's lighting estimation. Is there a particular effect you're trying to achieve?

**Reply by MathDestruction** [reply]
My team wanted to represent a building/hotel in AR where the the environmental lighting will change from day to night (Ambitious I know...). So I just wanted to get some feedback regarding the limitations of lighting in AR at the moment using model viewer

**Reply by elalish** [reply]
I that case I think you'll be better  served by sticking with WebXR mode and skipping lighting estimation entirely. Just change `environment-image` from a day HDR to a night HDR; you can even do that during an AR session, which is a big advantage of WebXR staying in the browser.

**Reply by MathDestruction** [reply]
Sorry just want to clarify.

"you can even do that during an AR session", is this *can't? Since it sounds like from other posts AR view, that is using SceneViewer or Quicklook, lighting cannot be adjusted?

Sorry if I'm misunderstanding somewhere along the line

**Reply by elalish** [reply]
That  why I recommended WebXR mode; it's the  only mode  where you can  control the lighting anyway, and it's also possible during a session. With Scene Viewer and Quick Look there is no customization.
